config/initializers/geocoder.rb

```diff
@@ -1,8 +1,8 @@
 redis = Redis.new(url: ENV['REDIS_CACHE_URL'])
-key = ENV.fetch('GOOGLE_GEOCODING_API_KEY', '')
+key = ENV.fetch('ORDNANCE_SURVEY_API_KEY', '')
 
 Geocoder.configure(
-  lookup: :google,
+  lookup: :os_names,
```

Comment:
Have we got enough testing to be confident that this replacement Just Works?

Reply:
Ah, I'd just typed that in the comment as you were typing this. There's tests in the gem, so these should be sufficient to give us confidence that switching providers will work  👍

Reply:
:+1: